build/plugins/annotations.html

```diff
@@ -0,0 +1,29 @@
+<!-- annotations plugin configuration -->
```

Comment:
I lean towards continuing to call this plugin "hypothesis" for backwards compatibility.

Reply:
I renamed the plugin file name back to `hypothesis.html`, but left the comment there to describe what it actually does for those who don't know. Is that okay?

I believe peoples' `hypothesis.html` file contents are going to have to be updated anyway because of added the configuration lines in it.